setup.cfg

```diff
@@ -21,6 +21,11 @@ per-file-ignores =
     mbed_targets/__init__.py:F401
 max-line-length = 160
 
+[coverage:run]
+omit =
+  # Top level entry point for mbed-tools, tested manually & in CI
+  mbed_targets/mbed_tools/cli.py
```

Comment:
Could use some advice about this, as I'm not sure what's the best way to approach coverage report in files like this. It doesn't seem reasonable to exercise them by automated testing.